frontend/lib/about/dev_settings_view.dart

```diff
@@ -179,6 +164,53 @@ class DevSettingsView extends StatelessWidget {
     );
   }
 
+  Future<void> _activateCard(BuildContext context, String base64qrcode) async {
+    final messengerState = ScaffoldMessenger.of(context);
+    final provider = Provider.of<UserCodeModel>(context, listen: false);
+    final client = GraphQLProvider.of(context).value;
+    final projectId = Configuration.of(context).projectId;
+    try {
+      final activationCode =
+          const ActivationCodeParser().parseQrCodeContent(const Base64Decoder().convert(base64qrcode));
+
+      final activationResult = await activateCode(
+        client: client,
+        projectId: projectId,
+        code: activationCode,
+        overwriteExisting: true,
+      );
+
+      if (activationResult.activationState == ActivationState.failed) {
+        throw QrCodeParseException("Code could not be activated from the server");
+      }
+      if (activationResult.activationState == ActivationState.didNotOverwriteExisting) {
+        throw QrCodeParseException("Did not overwrite existing QrCode.");
+      }
```

Comment:
Maybe we should invert the logic here:
if (activationResult.activationState !== ActivationState.success) { show error }

Reply:
so we should not consider ActivationState.did_not_overwrite_existing?

Reply:
It's just safer to invert the logic here.
If we ever add another entry to the enum in ActivationState, we might not be in the `success` state after checking for `didNotOverwriteExisting` and `failed`.

Reply:
i copied the logic from the normal activation workflow